### DISCUSSION TITLE
The questo for a (new) name

### DISCUSSION THREAD

**Discussion by audetto** [audetto/AppleWin#199] [General]
I have never liked the name of the main executable `sa2`, standing for `SDL Apple 2`.

Since I am building a snapcraft package, I have decided to rethink it.

My current preference is for

`ya2e`: Yet Another 2 Emulator

Why `y`? Because `x` is already abused to often.
Alternatives include `Classic`, `Virtual` or modifications of the above like `a2xe`, but so far my preference is above.

**Comment by sh95014** [comment]
I feel like "yet another" should belong to newer emulators, not a port of a very old one and almost "default" one like AppleWin.

How about something like "AW4L", meaning AppleWin for Linux? Or even "AppleLin"?  😊

**Comment by xandark** [comment]
Yes, sa2 is a bit of a stretch to associate as an Apple II emulator. `ya2e` doesn't quite feel right.

How about `a2em4lin`? `aw4lin`?

**Comment by hasseily** [comment]
I think at this point sa2 will become applewin, or more like appleall

**Comment by audetto** [comment]
I guess applewin will be used one the code is totally merged.

Current ideas:

1. `slot6`
2. `sapl` from Simple-Apple
3. `mosl` from MOS + Linux
